pykern/pkcli/test.py

```diff
@@ -17,9 +17,10 @@
 
 
 SUITE_D = "tests"
-_COROUTINE_NEVER_AWAITED = re.compile("RuntimeWarning: coroutine .+ was never awaited")
+_COROUTINE_NEVER_AWAITED = re.compile(
+    ".+\.py:\d+: RuntimeWarning: coroutine .+ was never awaited", flags=re.MULTILINE
```

Comment:
This regex can be simpler: `(.+ coroutine \S+ was never awaited.*)`. If this becomes too loose, we can always make it stricter. The failure mode of a too loose regular expression is going to be a failing test. That's an important concept.